api/scripts/send-invitations-to-sco-organizations.js

```diff
@@ -12,6 +12,8 @@ const organizationInvitationService = require('../lib/domain/services/organizati
 const organizationRepository = require('../lib/infrastructure/repositories/organization-repository');
 const organizationInvitationRepository = require('../lib/infrastructure/repositories/organization-invitation-repository');
 
+const TAGS = ['JOIN_ORGA'];
```

Comment:
Titre du commit
Add tags argument in script send invitations orga  => Add tags argument in send invitations orga script

Reply:
Done